models/autoencoders/gumboltCaloCRBM.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, **kwargs):
         self._level_2_qubit_idxs = level_2_qubit_idxs
         
         # Initialize the DWave QPU sampler
-        self._qpu_sampler = DWaveSampler(solver={"topology__type":"chimera", "chip_id":"DW_2000Q_6"})
+        self._qpu_sampler = DWaveSampler(solver={"topology__type":"chimera", "chip_id":"DW_2000Q_6"}, token="DEV-a0d61b807af3ba1c527907fb1febfc2ad1fe0191")
```

Comment:
@Sehmimul Lol! Not a good idea to publish your DWave token on a public repo.

Reply:
Yeah lol. Sorry it was my mistake. I just reset my account token. So this token is not active anymore.